### PR TITLE
Add documentation for CRI container metrics

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by yujuhong** [comment]
Ping for review! @Random-Liu @dchen1107

File: contributors/devel/cri-container-stats.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    // equal CapacityBytes - AvailableBytes.
+    UInt64Value used_bytes = 3;
+    // InodesUsed represents the inodes used by the images.
+    // This may not equal InodesCapacity - InodesAvailable because the underlying
+    // filesystem may also be used for purposes other than storing images.
+    UInt64Value inodes_used = 4;
+}
+```
+
+There are four categories or resources: CPU, memory, and filesystem. Each of
```
Comment (feiskyer) [inline contributors/devel/cri-container-stats.md]: s/four/three

**Pull request merged**